mock_provider/Dockerfile

```diff
@@ -4,8 +4,10 @@ FROM alpine:latest
 RUN apk add jq
 #Copying OAS spec file in container
 COPY "gp-connect-user-permissions.json" ./
-#Copyign HeslthStatus OAS file in container
+#Copyign HealthStatus OAS file in container
 COPY "HealthStatusEndpoint.json" ./
+#Copying whole components directory , to get inline examples and schemas
+COPY ./components ./components
```

Comment:
Like my first comment. Just copy the entire `specification` directory here and this dockerfile will be the same for all projects.

Reply:
Done